src/components/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Footer from './Footer';
+import Header from './Header';
+import 'normalize.css';
+
+import Photoalbum from './Photoalbum';
+import 'normalize.css';
+
+
+function App() {
+  // const cities = [portland, austin];
```

Comment:
Right here is where you'd want to initialize those cities so that you can pass them down to the different components. 
```suggestion
  // const cities = [portland, austin, LA, NYC, etc];
```

Then you want to pass them down to `Photoalbum`